### ISSUE TITLE
Reflection metadata is added to referencing assemblies

### ISSUE THREAD

**Issue by yuvaltz** [bridgedotnet/Bridge#2183]
### Actual
When referencing an assembly, its reflection metadata is being added to the generated code.

### Expected
Reflection metadata can be defined only once.

### Steps To Reproduce

* **ClassLibrary1**, **ClassLibrary2** are separate assemblies.
* **ClassLibrary1.Class1** has reflection metadata.
* **ClassLibrary2** is referencing **ClassLibrary1**.

```csharp
namespace ClassLibrary1
{
    [Bridge.Reflectable(true)]
    public class Class1
    {
    }
}
```

```csharp
namespace ClassLibrary2
{
    public class Class2
    {
        private ClassLibrary1.Class1 class1;
    }
}
```

Generated code for **ClassLibrary1**
```javascript
Bridge.assembly("ClassLibrary1", function ($asm, globals) {
    "use strict";

    Bridge.define("ClassLibrary1.Class1");

    var $m = Bridge.setMetadata,
        $n = [ClassLibrary1];
    $m($n[0].Class1, function () { return {"m":[{"a":2,"isSynthetic":true,"n":".ctor","t":1,"sn":"ctor"}]}; });
});
```

Generated code for **ClassLibrary2**, contains (some of) **ClassLibrary1.Class1** metadata
```javascript
Bridge.assembly("ClassLibrary2", function ($asm, globals) {
    "use strict";

    Bridge.define("ClassLibrary2.Class2", {
        class1: null
    });

    var $m = Bridge.setMetadata,
        $n = [ClassLibrary1];
    $m($n[0].Class1, function () { return {"m":[{"a":2,"n":".ctor","t":1,"sn":"ctor"}]}; });
});
```

Solution with the above configuration - [BridgeMetadataTest.zip](https://github.com/bridgedotnet/Bridge/files/655773/BridgeMetadataTest.zip)

**Comment by geoffreymcgill** [comment]
A pull-request has been submitted with a fix for this issue. We expect to include in the 15.7.0 release. After review, the pr will be merged into the **master** branch asap.
